www/js/services.js

```diff
@@ -0,0 +1,38 @@
+angular.module('open311.services', [])
+.factory('API', function($http, $httpParamSerializer, $q) {
+
+    var recentCasesData = [];
+
+    var getRequests = function(lat, long) {
+      var deferred = $q.defer();
+
+      var params = $httpParamSerializer({
+          lat: lat,
+          long: long 
+      });
+
+      $http.get('https://seeclickfix.com/open311/v2/requests.json?' + params)
```

Comment:
We're going to be building out more API functions soon, so move the root of this URL to it's own variable:

```js
var API_ENDPOINT = "https://seeclickfix.com/open311/v2";
// ...
$http.get(API_ENDPOINT + "requests.json?' + params)
```